src/scrape_up/imdb1/box_office.py

```diff
@@ -0,0 +1,134 @@
+    | `.top_movies()`    | Returns the top box office movies, weekend and total gross and weeks released |
+
+    """
+
+    def __init__(self):
+        self.__scrape_page()
+
+    def __scrape_page(self):
+        try:
+            url = "https://www.imdb.com/chart/boxoffice/?ref_=hm_cht_sm"
+            req = Request(url, headers={"User-Agent": "Mozilla/5.0"})
+
+            webpage = urlopen(req).read()
+            self.page_soup = soup(webpage, "html.parser")
+
+        except:
+            return None
+
+    def top_movies(self):
+
+        """
+        Create an instance of `BoxOffice` class
+
+        ```python
+        boxoffice = BoxOffice()
+        boxoffice.top_movies()
+        ```
+
+        Return\n
+        ```js
+        [
+            {
+                "Movie Name": "Barbie", 
+                "Weekend Gross": "$53M", 
+                "Total Gross": "$459M", 
+                "Weeks released": "3"
+            }, 
+            {
+                "Movie Name": "The Meg 2: The Trench", 
+                "Weekend Gross": "$30M", 
+                "Total Gross": "$30M", 
+                "Weeks released": "1"
+            }, 
+            {
+                "Movie Name": "Oppenheimer", 
+                "Weekend Gross": "$29M", 
+                "Total Gross": "$229M", 
+                "Weeks released": "3"
+            }, 
+            {
+                "Movie Name": "Teenage Mutant Ninja Turtles: Mutant Mayhem", 
+                "Weekend Gross": "$28M", 
+                "Total Gross": "$43M", 
+                "Weeks released": "1"
+            }, 
+            {
+                "Movie Name": "The Haunted Mansion", 
+                "Weekend Gross": "$9M", 
+                "Total Gross": "$42M", 
+                "Weeks released": "2"
+            }, 
+            {
+                "Movie Name": "Sound of Freedom", 
+                "Weekend Gross": "$7M", 
+                "Total Gross": "$163M", 
+                "Weeks released": "5"
+            }, 
+            {
+                "Movie Name": "Mission: Impossible - Dead Reckoning Part One", 
+                "Weekend Gross": "$6.5M", "
+                Total Gross": "$151M", 
+                "Weeks released": "4"
+            }, 
+            {
+                "Movie Name": "Talk to Me", 
+                "Weekend Gross": "$6.3M", 
+                "Total Gross": "$22M", 
+                "Weeks released": "2"
+            }, 
+            {
+                "Movie Name": "Indiana Jones and the Dial of Destiny", 
+                "Weekend Gross": "$1.5M", 
+                "Total Gross": "$171M", 
+                "Weeks released": "6"
+            }, 
+            {
+                "Movie Name": "Elemental", 
+                "Weekend Gross": "$1.2M", 
+                "Total Gross": "$148M", 
+                "Weeks released": "8"
+            }
+        ]
+        
```

Comment:
After first object in a list, enter `...` after it instead of mentioning all the objects in the list.